deployment.rb

```diff
@@ -218,7 +218,7 @@ def set_slogger_for_test(slogger)
   end
 
   def slog(params)
-    return unless slog_token
+    return unless slog_token && Gatekeeper.allows('activityMonitor', default: true)
```

Comment:
This check couples `slog` to the activity monitor functionality, which should be fine (in practice activity monitor is the only thing that uses the slog library as far as I know) but maybe worth leaving a note somewhere about this coupling.